packages/pointer_interceptor/pointer_interceptor_ios/example/pubspec.yaml

```diff
@@ -0,0 +1,26 @@
+name: pointer_interceptor_ios_example
+description: "Demonstrates how to use the pointer_interceptor_ios plugin."
+# The following line prevents the package from being accidentally published to
+# pub.dev using `flutter pub publish`. This is preferred for private packages.
+publish_to: 'none' # Remove this line if you wish to publish to pub.dev
+
+environment:
+  sdk: '>=3.1.0-134.0.dev <4.0.0'
```

Comment:
We should make this just 3.1.0

And we should have a Flutter constraint as well since it's a Flutter package.